README.md

```diff
@@ -141,6 +143,7 @@ Contribute adding resources or providing feedback through Pull Requests, Issues
 - [Tigermoth](https://github.com/pmprog/TigermothGBA) - Bullet hell game, developed for the GBAJam 2021
 - [Duster](https://github.com/redthing1/duster) - A sleek strategy board game for the gba 
 - [OpenLara](https://github.com/XProger/OpenLara) - Classic Tomb Raider open-source engine
+- [SS6PlayerForGBA](https://github.com/laqieer/SS6PlayerForGBA) - 2D skeletal animation engine.
```

Comment:
I don't see an english description/readme here..

Reply:
moved to `NON_ENG.md`